### ISSUE TITLE
Unable to create Collection (No community found)

### ISSUE THREAD

**Issue by ericlmontag** [DSpace/dspace-angular#659]
Even though I created a Community, I am unable to create a Collection within it.
This is the Community I created: 
Permanent URI for this community http://localhost:3000/handle/123456789/1

![Screenshot_2020-04-27 DSpace Angular Universal](https://user-images.githubusercontent.com/53164928/80413646-a1cf4f00-8884-11ea-9e03-9554359f26e0.png)

**Comment by ericlmontag** [comment]
ds-dso-selector (Create Collection Form. Does not return list of Communities.)
http://localhost:8080/server/api/discover/search/objects?dsoType=COMMUNITY

ds-top-level-community-list (Home page Community list. Does return list of Communities.) 
http://localhost:8080/server/api/core/communities/search/top

**Comment by tdonohue** [comment]
@ericlmontag : Thanks for reporting this! I am able to reproduce this.  It looks like it may actually be a bug on the **backend** (REST API), as I believe that first path *should return* a list of Communities.

For example, in the Create a New Item screen (which is similar), the list of Collections is retrieved via:
http://localhost:8080/server/api/discover/search/objects?dsoType=COLLECTION

But, you are correct, that same URL with `dsoType=COMMUNITY` seems to return no results (which is unexpected).

**Comment by tdonohue** [comment]
I've retested this on the latest `master` and I'm no longer able to reproduce this.  Keep in mind the application now runs on port 4000 by default cause of #625 (switch to Angular CLI).  I had previously tested this on an old version of `master` (prior to that CLI switch) and I saw the same behavior as reported by @ericlmontag .  However, I cannot reproduce it anymore, so I believe this may now be fixed.
